public/client.js

```diff
@@ -92,16 +84,16 @@ const appendContentInfo = (room_name, online, data_joined) => {
                                 </div>
                                 <div class='chat active-chat' data-chat='person1'></div>
                                 <div class="input-group write">
-                                  <textarea type="text" class="textarea form-control" placeholder="Message to ${room_name}..." data-active="lobby" rows="2"></textarea>
-                                  <div class="input-group-append">
-                                    <button class="btn smiley text-primary" type="button">
+                              <textarea type="text" class="textarea form-control" placeholder="Message to ${room_name}..." data-active="lobby" rows="2"></textarea>
+                              <div class="input-group-append">
+                                      <button class="btn smiley text-primary" type="button">
                                       <i class="far fa-smile icon"></i>
                                       ${emobox}
-                                    </button>
-                                    <button class="btn send text-primary" type="button">
+                                      </button>
+                                      <button class="btn send text-primary" type="button">
                                       <i class="fas fa-paper-plane icon"></i>
-                                    </button>
-                                  </div>
+                                      </button>
+                              </div>
```

Comment:
@sesamechicken you need to follow `.editorconfig` strictly, and if there is some ambiguity there please let me know or update that file also.

Reply:
I didn't go against the grain on the editor config. Again, there seems to be some misunderstanding surrounding linting/`eslint` and editor config. Happy to chat about it, but this is it for me on this one.